datamover/pkg/azure/blob/blobmover.go

```diff
@@ -52,6 +52,9 @@ func handleAzureBlobErrors(err error) error {
 			case string(azblob.StorageErrorCodeAuthenticationFailed):
 				log.Log("azure error: permission denied.")
 				return errors.New(DMERR_NoPermission)
+			case string(azblob.StorageErrorCodeContainerNotFound):
+				log.Log("azure error: container not found.")
+				return errors.New(DMERR_BucketNotFound)
```

Comment:
It seems this error is defined, but not used, so it seems not necessary.

Reply:
Using this we will get the specific error else it shows big error so this will allow us to detect easily that bucket is not available instead of big error message for Azure. Thanks